doc/user_guide/interactions.rst

```diff
@@ -504,3 +664,32 @@ for ideas about how they can be applied to more interesting charts.
 For more information on how to fine-tune selections, including specifying other
 mouse and keystroke options, see the `Vega-Lite Selection documentation
 <https://vega.github.io/vega-lite/docs/selection.html>`_.
+
+Limitations
+~~~~~~~~~~~
+
+Some possible use cases for the above interactivity are not currently supported by Vega-Lite, and hence are not currently supported by Altair.  Here are some examples.
+
+1. If we are using a ``selection_point``, it would be natural to want to return information about the chosen data point, and then process that information using Python.  This is not currently possible (and as of December 2021 it does not seem likely to become possible any time soon), so any data processing will have to be handled using tools such as ``transform_calculate``, etc. You can follow the progress on this in the following issue: https://github.com/altair-viz/altair/issues/1153.
```

Comment:
Should we mention that streamlit has some tools for this kind of thing?

Reply:
I believe [this was never implemented in streamlit](https://github.com/streamlit/streamlit/issues/2263) and only the proof of principle that domoritz created exists as of now. [It is possible in the master branch of panel as mentioned here](https://github.com/altair-viz/altair/issues/1153#issuecomment-996170916), but they have not had a release yet. I am planning to write something short about integrating with dashboard solutions [building on my answer here](https://stackoverflow.com/a/67428170/2166823) and will make sure to include this when there is a new release.